gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -15,8 +15,10 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     }
 
     private AndroidCommandPluginExtension createExtension() {
-        def project = ProjectBuilder.builder().build()
+	def projectDir = new File('..')
+        def project = ProjectBuilder.builder().withProjectDir(projectDir).build()
```

Comment:
Isn't there a better solution that works also when the project structure changes?